main.tf

```
@@ -17,6 +17,16 @@ module "subscriptions" {
   ]
 }
 
+# Manage Control status
+module "standards_control" {
+  source   = "./modules/standards_control"
+  for_each = { for control in var.standards_control : control.name => control }
+
+  standards_control_arn = each.key
```

oh, since we changed the key, this needs to use the arn value:

```suggestion
  standards_control_arn = each.value.arn
```